spec/coverage_helper_spec.rb

```diff
@@ -0,0 +1,19 @@
+require "spec_helper"
+
+if ENV['CI']
+  require 'active_support/core_ext/kernel/reporting'
+
+  describe "CoverageHelper" do
+    it "should require the world" do
+      %w(app gems lib spa_ui).each do |path|
+        Dir.glob(Rails.root.join(path, "**", "*.rb")) do |file|
+          next if file.to_s.include?("/spec/") || file.to_s.include?("/test/") || file.to_s.include?("test.rb")
```

Comment:
Wait, isn't `file` a String here?  Dir.glob will return an array of strings.

Note, I didn't realize you can do something similar to String#include with Pathnames:

`next if file.fnmatch?("*/spec/*") || file.fnmatch?("*/test/*") || file.fnmatch?("*/test.rb")`